Tools/Scripts/webkitpy/w3c/test_importer.py

```diff
@@ -209,7 +209,11 @@ def __init__(self, port, test_paths, options):
         }
 
     def do_import(self):
-        if not self.source_directory:
+        if self.source_directory:
+            source_path = Path(self.source_directory) / 'web-platform-tests'
```

Comment:
ha! my use of Pathlib seems incompatible with the tests

in short
```
  TypeError: 'PosixPath' object is not subscriptable
```

In details: 
```
[2029/2630] webkitpy.w3c.test_importer_unittest.TestImporterTest.test_crash_test_with_resource_file erred:
  Traceback (most recent call last):
    File "/Users/karlcow/code/OpenSource/Tools/Scripts/webkitpy/w3c/test_importer_unittest.py", line 470, in test_crash_test_with_resource_file
      fs = self.import_directory(['-s', '/home/user/wpt', '-d', '/mock-checkout/LayoutTests/imported/w3c/web-platform-tests'], FAKE_FILES, ['web-platform-tests/css/css-images'])
    File "/Users/karlcow/code/OpenSource/Tools/Scripts/webkitpy/w3c/test_importer_unittest.py", line 128, in import_directory
      importer.do_import()
    File "/Users/karlcow/code/OpenSource/Tools/Scripts/webkitpy/w3c/test_importer.py", line 214, in do_import
      git = self.test_downloader().git(source_path)
    File "/Users/karlcow/code/OpenSource/Tools/Scripts/webkitpy/w3c/test_downloader.py", line 102, in git
      return Git(test_repository, None, executive=self._host.executive, filesystem=self._filesystem)
    File "/Users/karlcow/code/OpenSource/Tools/Scripts/webkitpy/common/checkout/scm/git.py", line 65, in __init__
      SCM.__init__(self, cwd, **kwargs)
    File "/Users/karlcow/code/OpenSource/Tools/Scripts/webkitpy/common/checkout/scm/scm.py", line 68, in __init__
      self.checkout_root = self.find_checkout_root(self.cwd)
    File "/Users/karlcow/code/OpenSource/Tools/Scripts/webkitpy/common/checkout/scm/git.py", line 133, in find_checkout_root
      checkout_root = self._filesystem.join(path, checkout_root)
    File "/Users/karlcow/code/OpenSource/Tools/Scripts/webkitpy/common/system/filesystem_mock.py", line 218, in join
      if comp[0] == sep:
  TypeError: 'PosixPath' object is not subscriptable
  
```

Reply:
fixed.